OpenKeychain/src/test/java/org/sufficientlysecure/keychain/pgp/SplitUserIdTest.java

```diff
@@ -130,4 +130,12 @@ public void splitUserIdWithCommentShouldReturnComment() throws Exception {
         Assert.assertEquals("this is a comment", info.comment);
     }
 
-}
\ No newline at end of file
+    @Test
+    public void splitUserIdWithInvalidEmailShouldReturnEmail() {
```

Comment:
The function name does not match with the test. I would propose the following test:
```
    @Test
    public void splitUserIdWithEmailWithoutTldShouldReturnNameAndEmail() {
        OpenPgpUtils.UserId info = KeyRing.splitUserId("Max Mustermann <max@localhost>");
        Assert.assertEquals("Max Mustermann", info.name);
        Assert.assertEquals("max@localhost", info.email);
        Assert.assertNull(info.comment);
    }
```

Reply:
Done